core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1238,6 +1241,13 @@ private void unParseRegexpLike(SqlWriter writer, SqlCall call, int leftPrec, int
     writer.endFunCall(ifFrame);
   }
 
+  private void unparseShiftLeft(SqlWriter writer, SqlCall call) {
+    call.operand(0).unparse(writer, 0, 0);
+    writer.print(SHIFTLEFT);
```

Comment:
please use  writer.print(SHIFTLEFT+" ") instead of calling function twice.

Reply:
done